home/management/commands/broken_links_clean_up.py

```diff
@@ -0,0 +1,57 @@
+from django.core.management.base import BaseCommand
+
+from home.models import ContentPage, OrderedContentSet
+
+
+class Command(BaseCommand):
+    help = "Clean broken links for pages"
+
+    def handle(self, *args, **options):
+        content_page_links = []
+        ordered_content_links = []
+
+        # Only query live pages
+        pages = ContentPage.objects.live()
+
+        for page in pages:
+            if page.related_pages:
+                # Get related pages add it to the list if it is linked to a page that no longer exist
+                for rp in page.related_pages:
+                    if rp.value is not None:
```

Comment:
I'm not 100% sure how this works, but I think we should have a test for it anyway. My concern here is that `rp.value` still contains the ID to the related page, even when the page doesn't exist, and we have to actually lookup to see if a page with that ID still exists.

Reply:
@rudigiesler it is possible that there is something that i'm doing wrong, page.related returns a length of 2 if one is deleted then rp.value for the one that is deleted is empty. But i will relook to confirm.

Reply:
Ah okay, maybe having a test to confirm will be best then? I could be wrong and `rp.value` does the database lookup for us under the hood